language/move-vm/runtime/src/loader.rs

```diff
@@ -1029,10 +1115,23 @@ impl Loader {
         module: &CompiledModule,
         bundle_verified: &BTreeMap<ModuleId, CompiledModule>,
         data_store: &impl DataStore,
-        visited: &mut BTreeSet<ModuleId>,
+        visiting: &mut BTreeSet<ModuleId>,
         allow_dependency_loading_failure: bool,
         dependencies_depth: usize,
     ) -> VMResult<()> {
+        let cache_key = (data_store.link_context(), module.self_id());
+
+        // If this module is already in the "verified dependencies" cache, then no need to check it
+        // again -- it has already been verified uner
```

Comment:
"uner"?

Reply:
Yeah... not sure what I meant here (looks like I trailled off mid-sentence, but I think that comment is fine stopping before there).